inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/lrn.cpp

```diff
@@ -11,20 +11,76 @@
+                                           ::testing::Values(bias),
+                                           ::testing::Values(size),
+                                           ::testing::ValuesIn(axes),
+                                           ::testing::ValuesIn(netPrecisions),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(std::vector<size_t>({10, 16})),
+                                           ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                        LrnLayerTest::getTestCaseName);
+
+} // namespace LRN2D
+
+namespace LRN3D {
+
+const std::vector<std::vector<int64_t>> axes = {{1}, {2}};
+
+INSTANTIATE_TEST_CASE_P(smoke_LrnCheck_3D, LrnLayerTest,
+                        ::testing::Combine(::testing::Values(alpha),
+                                           ::testing::Values(beta),
+                                           ::testing::Values(bias),
+                                           ::testing::Values(size),
+                                           ::testing::ValuesIn(axes),
+                                           ::testing::ValuesIn(netPrecisions),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(std::vector<size_t>({6, 10, 16})),
+                                           ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                        LrnLayerTest::getTestCaseName);
+
+} // namespace LRN3D
+
+namespace LRN4D {
+
+const std::vector<std::vector<int64_t>> axes = {{1}, {2, 3}};
+
+INSTANTIATE_TEST_CASE_P(smoke_LrnCheck_4D, LrnLayerTest,
+                        ::testing::Combine(::testing::Values(alpha),
+                                           ::testing::Values(beta),
+                                           ::testing::Values(bias),
+                                           ::testing::Values(size),
+                                           ::testing::ValuesIn(axes),
+                                           ::testing::ValuesIn(netPrecisions),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(std::vector<size_t>({10, 10, 3, 8})),
+                                           ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                        LrnLayerTest::getTestCaseName);
+
+} // namespace LRN4D
+
+namespace LRN5D {
+
+const std::vector<std::vector<int64_t>> axes = {{1}, {2, 3, 4}};
```

Comment:
Does it makes sense to cover cases with non-default axes order like: 3,4,2?

Reply:
expanded